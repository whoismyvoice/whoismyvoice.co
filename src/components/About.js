import React from 'react'
import { Link } from 'react-router'

const About = React.createClass({
  render() {
    return <div className="page-block bright-red">
      <p className="pageIntroduction">
      	This website is unaffiliated with any government party or entity, nor are we a media outlet. Its objective is to provide comprehensive, actionable data to constituents who have a desire to better understand and contact the decision makers representing them.
      	<br /><br /> 
		    This website was created as a proactive in-house exercise by employees of Siberia, a global design and engineering firm. 
		    <br /><br />
		    Please send any comments or fact checks <Link className="strike-out" to="#">here</Link>.
		    <br /><br />
		    Please send any Business or Media inquiries <Link className="strike-out" to="#">here</Link>. 
      </p>
    </div>;
  }
});

export default About;
