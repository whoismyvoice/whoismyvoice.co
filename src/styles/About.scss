@import '_variables';
@import '_transitions';

.about {
  color: $white;
  background-color: $light-white;

  &:hover {
    color: $blue;
  }
}
