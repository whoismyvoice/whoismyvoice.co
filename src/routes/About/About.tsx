import * as React from 'react';

// Components
import FadedBG from '../../components/FadedBg';

// Assets
import '../../styles/About.scss';
import siberiaLogo from '../../img/siberia_logo.png';

class About extends React.Component {
  render(): JSX.Element {
    return (
      <div className="page-block light-white">
        <FadedBG color="white" />
        <div className="page-wrapper">
          <h2>About</h2>
          <p className="page-introduction">
            This website is unaffiliated with any government or lobbying party
            or entity, nor is it represented by a media outlet. Its objective is
            to provide easy access to comprehensive, actionable data to
            constituents who have a desire to better understand & reach the
            decision makers who represent them.
            <br />
            <br />
            This website was created as a proactive, in-house exercise by the
            team at Siberia.
            <br />
            <br />
            Please send any comments or fact checking{' '}
            <a
              href="mailto:q%75est&#105;%6f&#110;&#115;@w%68o%69s%6dyvoice%2e%63o?subject=whoismyvoivce.co%20Question"
              className="strike-out"
            >
              questions [at] whoismyvoice [dot] co
            </a>
            .
            <br />
            <br />
            Please send any business or media inquiries{' '}
            <a
              href="mailto:%63onta%63%74%40w%68%6fi%73%6d%79%76&#111;i%63%65&#46;%63&#111;?subject=whoismyvoivce.co%20Inquiry"
              className="strike-out"
            >
              contact [at] whoismyvoice [dot] co
            </a>
            .<br />
            <br />
            <a
              href="http://siberia.io"
              target="_blank"
              rel="noopener noreferrer"
            >
              <img alt="Siberia" src={siberiaLogo} width="40" />
            </a>
          </p>
        </div>
      </div>
    );
  }
}

export default About;
