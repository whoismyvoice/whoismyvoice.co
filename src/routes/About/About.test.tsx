import * as React from 'react';
import * as ReactDOM from 'react-dom';
import * as renderer from 'react-test-renderer';
import About from './About';

jest.mock('mixpanel-browser');

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<About />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders correctly', () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
