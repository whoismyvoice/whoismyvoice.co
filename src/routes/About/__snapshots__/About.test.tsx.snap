// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="page-block light-white"
>
  <div
    className="faded-bg faded-white hide"
  />
  <div
    className="page-wrapper"
  >
    <h2>
      About
    </h2>
    <p
      className="page-introduction"
    >
      This website is unaffiliated with any government or lobbying party or entity, nor is it represented by a media outlet. Its objective is to provide easy access to comprehensive, actionable data to constituents who have a desire to better understand & reach the decision makers who represent them.
      <br />
      <br />
      This website was created as a proactive, in-house exercise by the team at Siberia.
      <br />
      <br />
      Please send any comments or fact checking
       
      <a
        className="strike-out"
        href="mailto:questions@whoismyvoice.com?Subject=WhoIsMyVoice.com Question"
      >
        here
      </a>
      .
      <br />
      <br />
      Please send any business or media inquiries
       
      <a
        className="strike-out"
        href="mailto:contact@whoismyvoice.com?Subject=WhoIsMyVoice.com Contact"
      >
        here
      </a>
      .
      <br />
      <br />
      <a
        href="http://siberia.io"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Siberia"
          src={undefined}
          width="40"
        />
      </a>
    </p>
  </div>
</div>
`;
